common/presenters/population-table/locations-to-population-table-component.js

```diff
@@ -0,0 +1,70 @@
+const { parseISO, isValid } = require('date-fns')
+
+const tablePresenters = require('../table')
+
+const { dayConfig, establishmentConfig } = require('./population-table-config')
+
+function locationsToPopulationTable({
+  isSortable = false,
+  query,
+  focusDate,
+  startDate,
+} = {}) {
+  if (!isValid(startDate)) {
+    startDate = parseISO(startDate)
+  }
+
+  if (!isValid(startDate)) {
+    startDate = new Date()
+  }
+
+  const tableConfig = [
+    {
+      ...establishmentConfig({}),
+    },
+    {
+      ...dayConfig({
+        focusDate: focusDate,
+        baseDate: startDate,
+        baseOffset: 0,
+      }),
+    },
+    {
+      ...dayConfig({
```

Comment:
I don't think you need to use the spread operator here and you can just do:

```javascript
[
  ...
  dayConfig({
    focusDate: focusDate,
    baseDate: startDate,
    baseOffset: 1,
  }),
  dayConfig({
    focusDate: focusDate,
    baseDate: startDate,
    baseOffset: 2,
  }),
  dayConfig({
    focusDate: focusDate,
    baseDate: startDate,
    baseOffset: 3,
  }),
  ...
]
```

Reply:
Replaced the spread with the loop concept from above